### PR TITLE
fix: sorting folders

### LINKED ISSUE #81: Update repo - sort folders correctly
Just a visual correction in the repo, sorting the folders correctly, so it will follow an order:

"01-getting-started"
"02-variables"
...
"10-preprocessors"
...

I'm think that would be great as the project grows to find faster the subject you want.

### PR DESCRIPTION
**Pull request by ropoko**
### 2. Description
The folders name were renamed to resolve the sorting error.

### 3. Fill in checklist by marking [x]

- [x] I've read the CONTRIBUTING.md
- [x] I was assigned to this issue
- [ ] My code is formatted using required linting
- [x] I've run my code locally and checked it works

No code was written.

### CODE REVIEW

**Comment by sabderemane** [comment]
Hi @ropoko , thank you for the suggestion ! I'll suggest also to review the README also because links will be broken after that.

**Comment by ropoko** [comment]
README fixed!

File: README.md [hunk 1]
```diff
@@ -19,31 +19,31 @@ Each file contains:
...
-| User input in arrays          | [userinput-in-arrays.c](6-arrays/userinput-in-arrays.c)       |
+| User input in arrays          | [userinput-in-arrays.c](06-arrays/userinput-in-arrays.c)       |
 | **Structures**                |
-| Declaring structures          | [declaring-structures.c](9-structures/declaring-structures.c) |
-| Typedefs                      | [typedefs.c](9-structures/typedefs.c)                         |
+| Declaring structures          | [declaring-structures.c](09-structures/declaring-structures.c) |
+| Typedefs                      | [typedefs.c](09-structures/typedefs.c)                         |
 | **Common built-in functions** |
-| Scanf                         | [scanf.c](11-common-built-in-functions/scanf.c)               |
+| Scanf                         | [scanf.c](011-common-built-in-functions/scanf.c)               |
```
Comment (sabderemane) [inline README.md:45]:
```suggestion
| Scanf                         | [scanf.c](11-common-built-in-functions/scanf.c)               |
```

**Review by sabderemane — CHANGES_REQUESTED** [review]

**Comment by ropoko** [comment]
sorry, I closed it by accident

**Review by sabderemane — APPROVED** [review]

**Comment by sabderemane** [comment]
Nevermind, it's good to me I merged!

**Pull request merged**